webui/react/src/components/ModelRegistry.settings.ts

```diff
@@ -43,7 +43,7 @@ export const isOfSortKey = (sortKey: React.Key): sortKey is V1GetModelsRequestSo
   return sortKeys.includes(String(sortKey));
 };
 
-export interface Settings extends InteractiveTableSettings {
+export interface ReducedSettings extends InteractiveTableReducedSettings {
```

Comment:
nit: consider `Omit<InteractiveTableSettings, 'tableOffset'>` instead -- the interactive table doesn't need to know that the table offset is coming from somewhere else